### PR TITLE
fixes in api

### PR DESCRIPTION
**Pull request by FdjNL**
fixed the api url 
and updated the wraper so it also gets the new fields

### CODE REVIEW

**Comment by FdjNL** [comment]
why does it fail on the 'BeenPwned.Api.nuspec' ?

**Comment by jfversluis** [comment]
Whoops! Looks like something got mixed up 😄 fixed that now!

**Review by jfversluis — CHANGES_REQUESTED** [review]
Could you rename `Adres` to `Address` to match the rest? Also it looks like spaces and tabs indentations are now mixed. I don't really care which we use, as long as we pick one of both.

File: OverheidioAPI-NET/Models/Adress.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+
+namespace OverheidioApi.NET.Models
+{
+	public class Adress
```
Comment (jfversluis) [inline OverheidioAPI-NET/Models/Adress.cs]: Double d as well :)

**Review by jfversluis — APPROVED** [review]

**Pull request merged**